etc/config/csharp.defaults.properties

```diff
@@ -1,19 +1,20 @@
-compilers=dotnet6csharp:dotnet7csharp
+compilers=&csharp
 supportsBinary=true
 needsMulti=false
 compilerType=csharp
 defaultCompiler=dotnet7csharp
 
+group.csharp.compilers=dotnet6csharp:dotnet7csharp
```

Comment:
Ok, now that you're here, we'll need one more fix:

The config system first reads `.defaults` and then overrides with `.amazon`, which is great for some things, but can bite us if not done with care. To avoid future issues, can you change the id of the `.defaults` compilers to have a `def` postfix?
Same with the groups

Reply:
Done.